libkrb5/src/context.rs

```diff
@@ -297,6 +297,8 @@ impl Krb5Context {
         let mut auth_ctx = auth_context.auth_context;
         let mut ap_req_options: krb5_flags = (AP_OPTS_MUTUAL_REQUIRED | AP_OPTS_USE_SESSION_KEY) as i32;
 
+        /* Assemble the authenticator checksum field, as per RFC 4121, section 4.1.1.
+           We only use the Flags field to request service options from the server. */
         let code = unsafe {krb5_auth_con_set_req_cksumtype(self.context, auth_context.auth_context, GSS_CHECKSUM_TYPE)};
         krb5_error_code_escape_hatch(self, code)?;
 
```

Comment:
A 306-os sorba célszerű lenne a doksiban használt elnevezéseket felvenni akkor is, ha nem használjuk őket.
```
let binding_info: Vec<u8> = iter::repeat(0).take(16).collect();
        let mut checksum_data: Vec<u8> = [&16_u32.to_le_bytes(), binding_info.as_slice(), &checksum_flags.to_le_bytes()].concat();
```

Reply:
Done